### ISSUE TITLE
The stack is all wrong

### ISSUE THREAD

**Issue by chrisdev** [kaguillera/newspipeline#1]
Here's the stack we're were hoping to build in comparison with the Foundation Email
<img width="675" alt="email-production-stack" src="https://cloud.githubusercontent.com/assets/701689/14568947/cfaeac4a-0309-11e6-9cdf-aa8d4e68088e.png">

**Comment by chrisdev** [comment]
@kaguillera there is no reason for Django, Wagtail etc.

**Comment by kaguillera** [comment]
I was actually wondering if to delete the repository and start all over again or make changes (drastic) on this repo. What do you think?

**Comment by kaguillera** [comment]
For now I well leave it and do all my work in the workspace directory.

**Comment by chrisdev** [comment]
Maybe a fresh start is needed? You may want to base your library structure on pelican. The template layout stuff can be a theme

**Comment by kaguillera** [comment]
Ok. right now I am trying to see if I can install node.js in a virtualenv without having to use nodeenv

**Comment by chrisdev** [comment]
Follow the instruction from Foundation

**Comment by chrisdev** [comment]
http://foundation.zurb.com/emails/docs/sass-guide.html

**Comment by chrisdev** [comment]
https://nodejs.org/en/

**Comment by kaguillera** [comment]
yeah but that installs node.js globally I wanted to restrict it to a virtual env
